docs/references.rst

```diff
@@ -73,6 +77,11 @@ Glossary
     Ring Background
       Background estimation method typically used for image analysis.
 
+    RoI
+      Stands for "region of interest", and indicates the spatial region in which the
```

Comment:
Minor : for abbreviations we could introduce a uniform structure like:
```
RoI
    Short for "Region of Interest":

GTI
    Short for "Good Time Interval":
```
I think there a few other entries that could be adapted to such a scheme.